cfgov/cfgov/settings/base.py

```diff
@@ -767,7 +767,7 @@
     # Enable django-elasticsearch-dsl and disable haystack in regulations3k.
     "ELASTICSEARCH_DSL_REGULATIONS": [("boolean", False)],
     # Enable django-elasticsearch-dsl and disable haystack in paying_for_college.
-    "ELASTICSEARCH_DSL_PFC": [("boolean", False)],
+    "ELASTICSEARCH_DSL_PFC": [("boolean", True)],
```

Comment:
I think we should remove the flag, now that it's not being used.

Reply:
Agreed and I removed the DSL_PFC flag.